README.md

```diff
@@ -105,6 +105,12 @@ module.exports = function (grunt) {
 };
 ```
 
+or via the CLI
+
+```sh
+$ bower-ls js css
```

Comment:
Dang. I could have sworn I checked that. Another reason not to write PRs late at night, I guess.